.github/workflows/test.yml

```diff
@@ -1015,3 +1016,57 @@ jobs:
       - name: valgrind
         run: |
           nix develop -c zig build test-valgrind
+
+  build-freebsd:
+    name: Build on FreeBSD
+    needs: test
+    runs-on: namespace-profile-mitchellh-sm-systemd
+    strategy:
+      matrix:
+        release:
+          - "14.3"
+          # - "15.0" # disable until fixed: https://github.com/vmactions/freebsd-vm/issues/108
+    steps:
+      - name: Checkout Ghostty
+        uses: actions/checkout@08eba0b27e820071cde6df949e0beb9ba4906955 # v4.3.0
+
+      - name: Start SSH
+        run: |
+          sudo systemctl start ssh
+
+      - name: Set up FreeBSD VM
+        uses: vmactions/freebsd-vm@05856381fab64eeee9b038a0818f6cec649ca17a # v1.2.3
+        with:
+          release: ${{ matrix.release }}
+          copyback: false
+          usesh: true
+          prepare: |
+            pkg install -y \
```

Comment:
You might want to switch to the `latest` package set before installing deps like it is done in https://github.com/PackageKit/PackageKit/blob/c7508778a1ead262c425a305bd02a92ee2ab2cc8/.cirrus.yml#L7-L8